include/swift/Option/Options.td

```diff
@@ -619,6 +619,10 @@ def whole_module_optimization : Flag<["-"], "whole-module-optimization">,
   HelpText<"Optimize input files together instead of individually">,
   Flags<[FrontendOption, NoInteractiveOption]>;
 
+def batch_mode : Flag<["-"], "batch-mode">,
+  Flags<[FrontendOption, NoInteractiveOption]>,
```

Comment:
Please mark this HelpHidden for now. Also, *is* it a FrontendOption? Didn't @davidungar  already add one?

Reply:
@jrose-apple There is currently no other mention of batch mode in FrontendOptions.td or Options.td. There might be in one of the changesets that @davidungar has open.

I do not otherwise understand the question: do you want it to be added to FrontendOptions.td as well, here? Or Is it implied by being added to Options.td with the FrontendOption flag? I'm afraid I still don't really understand how the definitions in these two files, and the flags describing frontend-ness, relate.

(Will add HelpHidden, of course, good catch.)

Reply:
The FrontendOption flag makes it available to the frontend when in Options.td; it's a default flag in FrontendOptions.td. (You can see how that works up at the top of FrontendOptions.td.) I just wasn't sure whether we needed or wanted a batch mode flag for the frontend at this point, or whether David had already added one (in which case I *think* the option-parsing system would complain that there were two flags with the same name). If this is the correct thing then go for it.